<?xml version="1.0"?>
<launch>
	<!-- Parameters to pass to the ROS node ns="$(arg robot_name)"  ns="$(arg robot_name)"-->
	  

	<arg name="move_group_name" default="manipulator"/>
  	<arg name="robot_name" default="iiwa"/>  
	
	<group> 
		<node pkg="iiwa_grasp" name="env"  type="env_robot.py" output="screen"/>
		<param name="move_group_name" value="$(arg move_group_name)"/>  
	</group> 

</launch>
